site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -14581,10 +14581,9 @@ private void collectNewSiteInfo(SessionState state,
 							Section s = cms.getSection(providerSectionId);
 							if (s != null)
 							{
-								String sDescription = StringUtils.trimToNull(s.getDescription());
-								if (sDescription != null && !siteInfo.description.contains(sDescription))
+								if (!StringUtils.containsIgnoreCase(siteInfo.description,  s.getDescription()))
```

Comment:
I might add a one line comment in here about what this code is doing and why it's here but otherwise looks equivalent and safer than the original. Just looking at it I wouldn't know what is going on. It seems like it's saying if the site info description does not contain the section description then concat them together. This is what's happening right?